screen.css

```diff
@@ -1245,3 +1245,572 @@
+    padding: 150px 400px 150px 400px;
+  }
+
+  .inner-div {
+    padding-right: 30px;
+    padding-left: 20px;
+    width: 550px;
+    height: 100px;
+    margin-bottom: 100px;
+  }
+
+  .inner-div h2 {
+    color: white;
+    text-align: center;
+    font-size: 40px;
+    font-weight: 400;
+    padding-bottom: 0;
+  }
+
+  #inner-div {
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+  }
+
+  .form {
+    padding: 20px;
+  }
+
+  li input {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    font-family: Poppins, sans-serif;
+    border-radius: 8px;
+    width: 100%;
+    height: 50px;
+    padding-left: 10px;
+    color: #000;
+  }
+
+  li textarea {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    font-family: Poppins, sans-serif;
+    border-radius: 8px;
+    width: 100%;
+    height: 150px;
+    padding-left: 10px;
+    font-size: 15px;
+    line-height: 24px;
+    padding-top: 10px;
+  }
+
+  .inner-list {
+    display: flex;
+    flex-direction: column;
+    margin: auto;
+    width: 100%;
+  }
+
+  .inner-list .flex-item6 button {
+    margin-left: 150px;
+    margin-top: 1px;
+  }
+
+  .all-item:hover {
+    transform: scale(1, 0.8);
+  }
+
+  .desk-item a:hover {
+    color: #3d4ee4;
+  }
+
+  #portfolioPopup {
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgb(76, 78, 82, 0.7);
+    display: none;
+  }
+
+  ul.works-flex-item.titles-container {
+    margin: 10px;
+  }
+
+  .works-flex {
+    padding: 20px;
+    background: #fff;
+    margin: 50px 100px;
+    border-radius: 10px;
+  }
+
+  .work-image > img {
+    width: 100%;
+  }
+
+  .titles-item {
+    font-family: 'Poppins', sans-serif;
+    font-size: 16px;
+    color: lightslategray;
+    font-weight: 500;
+    list-style-image: url(../images/Counter.png);
+    padding-right: 25px;
+  }
+
+  .list-main {
+    padding-right: 25px;
+    list-style-type: none;
+    font-weight: 600;
+    font-size: 18px;
+    color: #344563;
+    line-height: 24px;
+  }
+
+  .works-flex-pop {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: baseline;
+    width: 100%;
+    margin-left: 10px;
+    font-size: 25px;
+  }
+
+  .project-tags {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .work-description {
+    width: 75%;
+  }
+
+  .project-tags > div {
+    margin-left: 10px;
+  }
+
+  .tags {
+    width: 25%;
+  }
+
+  .tags > ul {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    list-style: none;
+    margin-left: 5px;
+    margin-top: 15px;
+  }
+
+  .lang-group,
+  .lang-group-2 {
+    background: #ebebff;
+    border-radius: 8px;
+    font-size: 12px;
+    padding: 4px 12px 4px;
+    color: #6070ff;
+    margin-top: 5px;
+    line-height: 16px;
+  }
+
+  .btn-tags {
+    border-top: 1px solid rgb(236, 234, 234);
+    padding-top: 20px;
+  }
```

Comment:
Great job @Nowembabazi 🤝

- [ ] Kindly fix the responsiveness of your desktop popup. Your buttons seem to be overlapping each other at the moment. Kindly work on the responsiveness of your popup window.
![buttons desktop](https://user-images.githubusercontent.com/63917725/189387960-93ea4dcd-21c3-428a-b241-6e0d2b217b25.PNG)